rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -513,6 +513,29 @@ void getPendingFederatorPublicKeyOfType() {
             assertThat(bridgeSupport.getPendingFederatorPublicKeyOfType(0, FederationMember.KeyType.MST), is(mstKey.getPubKey()));
         }
 
+        @Test
+        void getProposedFederationAddress_whenFederationSupportReturnsEmpty_shouldReturnEmpty() {
+            // Act
+            var actualProposedFederationAddress = federationSupport.getProposedFederationAddress();
```

Comment:
This test file is called `BridgeSupportTest`. Should we call `bridgeSupport. getProposedFederationAddress()` instead?

Should tests be added to the `ProposedFederationTests` as well?

Reply:
Yes, updated. `ProposedFederationTests` I do not seem to find it?

Reply:
yeah it does not exist, we test proposed federation related things in FederationSupport tests